tests/core/test_utils.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 import unittest
 import os
+import pytest
```

Comment:
hi,

thanks for the tip.
I didn't understand something:

I did use make tests to run all the test in the project.
but how could I avoid importing pytest?

I have used it in the same file on line 266:
      with pytest.raises(ValueError) as excinfo:

so wasn't I supposed to import pytest?
is there another way to reference it?

Thank you!